### PR TITLE
Show project name instead of full project file path in Navigation Bar

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
image

This is what C# and VB do.

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs [hunk 1]
```diff
@@ -97,6 +97,8 @@ type internal FSharpBlockStructureService(checker: FSharpChecker, projectInfoMan
                 | Some parsedInput ->
                     let blockSpans =
                         Structure.getOutliningRanges (sourceText.Lines |> Seq.map (fun x -> x.ToString()) |> Seq.toArray) parsedInput
+                        // it's crucial to not return more than one BlockSpan, othewise Roslyn crashes Visual Studio when user hover
```
Comment (cartermp) [inline vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs]: Comment seems unrelated to this PR - is it?  I have no objection to adding this, just curious.

Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Structure/BlockStructureService.fs]: Yes, it's an unrelated comment, I forgot to add it into "fix structure service" PR.

**Comment by KevinRansom** [comment]
Thank you for taking care of this

Kevin
